site/content/in-dev/unreleased/configuring-polaris-for-production.md

```diff
@@ -125,6 +125,19 @@ polaris.realm-context.require-header=true
 This will cause Polaris to also return a `404 Not Found` response if the realm header is not present
 in the request.
 
+### Polaris Server Header
+
+Polaris can emit an informational `Server` HTTP response header using Quarkus' built-in header
+configuration. Add the following property to one of the supported configuration sources (for example,
+`application.properties`) to enable it with the Polaris version string:
+
+```properties
+quarkus.http.header."Server".value=Polaris/${quarkus.application.version}
+```
+
+If you prefer to scope the header to specific environments, only set the property for the desired
+profile (for example, `%prod`).
```

Comment:
```suggestion
```
I could imagine that "specific environment" is confusing for non-Quarkus affine users.